eng/mgmt/mgmtmetadata/iothub_resource-manager.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/iothub/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --tag=package-2020-03 --csharp-sdks-folder=C:\code\Azure\asfn1\sdk
```

Comment:
why is this tag removed?

Reply:
As specified elsewhere, if we don't specify a tag then it reverts to the default, as specified in the readme.md in the swagger repo. We want the default.

Previously we specified this to hard code an older version, to avoid the deserialization bug (that has now been fixed).